### PR TITLE
 Improve WebClip name extraction function 

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] #1405
[Problem] WebClip name extraction function is not working correctly with full urls
[Solution] Improve getWebClipName function

### CODE REVIEW

File: examples/mobile/HomeApp/js/app.js [hunk 1]
```diff
@@ -101,8 +101,10 @@
 	//TODO: provide mechanism for getting web clip name from webClip meta data
 	//		and separate from getting ID
 	function getWebClipName(webClip) {
-		webClip = webClip.replace("webclip/", "").replace("/", "");
-		return webClip;
+		const pos = webClip.lastIndexOf("/"),
```
Comment (TomaszLukawskiSam) [inline examples/mobile/HomeApp/js/app.js]: maybe regexp  will be better `.replace(/\/$/, "")`

Comment (lslachciak) [reply examples/mobile/HomeApp/js/app.js]: This is working properly
`/.*\//`
https://regex101.com/r/CRiluw/2

**Comment by lslachciak** [comment]
Used regular expression for name replace function

File: examples/mobile/HomeApp/js/app.js [hunk 2]
```diff
@@ -99,8 +99,8 @@
 	//TODO: provide mechanism for getting web clip name from webClip meta data
 	//		and separate from getting ID
 	function getWebClipName(webClip) {
-		webClip = webClip.replace("webclip/", "").replace("/", "");
-		return webClip;
+		// remove all text to the last \
+		return webClip.replace(/.*\//, "");
```
Comment (TomaszLukawskiSam) [inline examples/mobile/HomeApp/js/app.js:103]: Do you need "g" (as global regex extension) or not?

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Pull request merged**